src/api/app/views/layouts/webui2/webui.html.haml

```diff
@@ -9,6 +9,13 @@
     %title
       = "#{@pagetitle} - " if @pagetitle
       = home_title
+    %meta{ property: "og:title", content: "#{meta_title}"}/
```

Comment:
content: meta_title should do - no need to interpolate.

Reply:
Additionally to the title I would add `og:site_name`, with instance name.

Reply:
Added `og:site_name` and removed all the extra interpolation!